draft/spec/index.html

```diff
@@ -1278,9 +1278,11 @@ <h2>Moab in an OCFL Object</h2>
             OCFL conversion tool. This tool uses the Moab-versioning gem to extract deltas and digests
             of the Moab data directory for each Moab version and translate those into version state blocks
             in an OCFL inventory file, which would be placed in the root directory of the Moab object.
-            All extant files in the Moab version directories are tracked in the manifest block. The contents
-            of the Moab manifests directory are not tracked in the version state blocks; they are effectively
-            deleted from the OCFL object without compromising legacy access patterns.
+            The content of the /data directory in the Moab version directories (and thus, the bitstreams
```

Comment:
Change `cand I think we should drop the leading slash -- I'm really not sure what that means given that one already says that `data` is in the version directory

Reply:
Resolved in new commit